### ISSUE TITLE
Please Help Me

### ISSUE THREAD

**Issue by loverboy2000** [qH0sT/AndroSpy#191]
Can I Fix This Error
![image](https://user-images.githubusercontent.com/13707724/146041867-c5ffc90f-612a-47f3-97ef-f1c79871979d.png)

**Comment by rockpapersissors** [comment]
I have the same error on my laptop x86 because androspy builder doesn't support x86 PC's, it will only work on x64, is your pc x86 by any chance?

**Comment by rockpapersissors** [comment]
I have the same error on my laptop x86 but androspy builder on my x64 work fine ,it's because you didn't build task2.sln correctly I think

**Comment by rockpapersissors** [comment]
I already had this problem before and told Qh0st about but didn't care ,it's probably just because you didn't build task2.sln correctly, try rebuild it
![Capture](https://user-images.githubusercontent.com/88531648/146088772-a7c4206b-13df-461f-bd1b-b6e6b1a6cd4e.PNG)

**Comment by loverboy2000** [comment]
> I has the same problem and told Qh0st about but he didn't care ,it's probably just because you didn't build task2.sln correctly, try rebuild it,on my old laptop x86 it won't let me build it without errors ![Capture](https://user-images.githubusercontent.com/88531648/146088772-a7c4206b-13df-461f-bd1b-b6e6b1a6cd4e.PNG)

Do you have a version that works without problems now?

**Comment by rockpapersissors** [comment]
> > I has the same problem and told Qh0st about but he didn't care ,it's probably just because you didn't build task2.sln correctly, try rebuild it,on my old laptop x86 it won't let me build it without errors ![Capture](https://user-images.githubusercontent.com/88531648/146088772-a7c4206b-13df-461f-bd1b-b6e6b1a6cd4e.PNG)
> 
> Do you have a version that works without problems now?

Yes I have 2 pc, One x86 and x64, androspy builder work fine on x64 but not on the x86
